homeassistant/components/sensor/rmvtransport.py

```diff
@@ -77,20 +86,22 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
                 next_departure.get(CONF_TIME_OFFSET),
                 next_departure.get(CONF_MAX_JOURNEYS),
                 next_departure.get(CONF_NAME)))
-    add_entities(sensors, True)
+    async_add_entities(sensors, True)
 
 
 class RMVDepartureSensor(Entity):
     """Implementation of an RMV departure sensor."""
 
-    def __init__(self, station, destinations, directions,
-                 lines, products, time_offset, max_journeys, name):
+    def __init__(self, session, station, destinations, directions, lines,
+                 products, time_offset, max_journeys, name):
         """Initialize the sensor."""
+        self._session = session
```

Comment:
We don't need to store the session either since we just need to pass it to the instantiation of the RMV data in this method.

Reply:
Thank you Martin for pointing that out.